gui/applicationlist.cpp

```diff
@@ -222,15 +222,15 @@ bool ApplicationList::findDefaultWindowsEditor()
     if (checkAndAddApplication(windowsPath + "\\system32\\notepad.exe", "Notepad", "(file)"))
         foundOne = true;
 
-    QString regPath = "HKEY_CLASSES_ROOT\\Applications\\QtProject.QtCreator.pro\\shell\\Open\\command";
+    QString regPath = R"(HKEY_CLASSES_ROOT\Applications\QtProject.QtCreator.pro\shell\Open\command)";
```

Comment:
I am skeptic about this. Use raw strings when it is hard to read the escaped string.

Reply:
I think it also comes in handy when you need to copy and paste it. It's always a pain to add/remove the escaping when you need to put it into another tool.